src/mwi/get_ecmwf_seasonal_data.py

```diff
@@ -47,9 +47,12 @@ def main(download=True, process=True):
     if download:
         area = AreaFromShape(df_country_boundaries.buffer(3))
         ecmwf_forecast.download(country_iso3=COUNTRY_ISO3, area=area)
+        ecmwf_forecast.process(country_iso3=COUNTRY_ISO3)
```

Comment:
Would be good to document these arguments too. A bit confusing that the download section calls a `.process()` method, which isn't related to the boolean `process` which if `True` computes stats.

Reply:
Fair point. Changed the boolean name and added a few comments. Hope that makes it a bit more clear